pkg/volumes/builder.go

```diff
@@ -129,3 +129,16 @@ func buildVolumeAndMount(refType flowslatest.MountableType, refName string, volu
 		MountPath: "/var/" + volumeName,
 	}
 }
+
+func (b *Builder) insertOrReplace(vi *VolumeInfo) {
```

Comment:
why do we need that? Are we adding several times the same volume?

Reply:
While adding the tests on console plugin I faced issues with volumeInfos duplicated.
This solve the issues and ensure each volume is appened once.

https://github.com/netobserv/network-observability-operator/pull/329/commits/9f52e81b6f5ddc0ae528e58a57a69c345648eadc